src/main/java/org/sagebionetworks/web/client/presenter/ProfilePresenter.java

```diff
@@ -601,24 +602,21 @@ public void getFavorites() {
 			public void onSuccess(List<EntityHeader> result) {
 				if (filterType == ProjectFilterEnum.FAVORITES) {
 					// convert to Project Headers
-					if (result.size() == 0) {

```

Comment:
Show the FavoritesHelpPanel if no _Projects_ have been marked as being a favorite.